Documentation/openssl.md

```diff
@@ -20,6 +20,14 @@ If you will be running a highly-available control-plane consisting of multiple m
 
 The IP address of the Kubernetes API Service. The `K8S_SERVICE_IP` will be the first IP in the `SERVICE_IP_RANGE` discussed in the [deployment guide][deployment-guide]. The first IP in the default range of 10.3.0.0/24 will be 10.3.0.1. If the SERVICE_IP_RANGE was changed from the default, this value must be updated as well.
 
+<hr/>
+
+**WORKER_IP**=_no default_
+
+**WORKER_FQDN**=_no default_
+
+The IP addresses and fully qualifed hostnames of all worker nodes will be needed. In most cases these will be the main routable IP/hostname of the worker machines. These will be unique per worker; when you see them used below, consider it a loop and do that step for _each_ worker.
```

Comment:
Workers may not have a routable hostname. We may want to change this wording to reflect that it is technically optional.

Maybe something along the lines of: "The certificates generated for the worker nodes will need to reflect how requests will be routed to those nodes. In most cases this will be a routable IP and/or a routable hostname. These will be unique per worker..."